lib/shopify_cli/tasks/update_dashboard_urls.rb

```diff
@@ -1,18 +1,21 @@
+require 'set'
```

Comment:
Rubocop is complaining about this, I'll change it.

```suggestion
require "set"
```